### ISSUE TITLE
Not a Sunrise but a Galaxyrise

### ISSUE THREAD

**Issue by wireddown** [wireddown/ghpages-ghcomments#4]
This is the comment thread for [Not a Sunrise but a Galaxyrise](http://wireddown.github.io/ghpages-ghcomments/2015/01/12/not-a-sunrise-but-a-galaxyrise).

**Comment by wireddown** [comment]
This kid is in the ninth grade, Dude, and his father is—are you ready for this? —Arthur Digby Sellers. Uh, yeah. Probably a vagrant, slept in the car. Or perhaps just used it as a toilet, and moved on. D'ya have a good sarsaparilla?

**Comment by wireddown** [comment]
Yes, so we were informed. However, we must of course transmit the remains to you in a receptacle. I'm saying, Cynthia's Pomeranian. I'm looking after it while Cynthia and Marty Ackerman are in Hawaii. A way out west there was a fella, fella I want to tell you about, fella by the name of Jeff Lebowski. At least, that was the handle his lovin' parents gave him, but he never had much use for it himself.

This Lebowski, he called himself the Dude. Now, Dude, that's a name no one would self-apply where I come from. But then, there was a lot about the Dude that didn't make a whole lot of sense to me. And a lot about where he lived, likewise. But then again, maybe that's why I found the place s'durned innarestin'.

**Comment by wireddown** [comment]
Hey! This is a private residence, man! Every time a rug is micturated upon in this fair city, I have to compensate. Obviously you're not a golfer.

**Comment by wireddown** [comment]
You're going to enter a world of pain, son. We know that this is your homework. We know you stole a car. They call Los Angeles the City of Angels. I didn't find it to be that exactly, but I'll allow as there are some nice folks there. 'Course, I can't say I seen London, and I never been to France, and I ain't never seen no queen in her damn undies as the fella says. But I'll tell you what, after seeing Los Angeles and thisahere story I'm about to unfold —wal, I guess I seen somethin' ever' bit as stupefyin' as ya'd see in any a those other places, and in English too, so I can die with a smile on my face without feelin' like the good Lord gypped me.

**Comment by wireddown** [comment]
Please see him, Jeffrey. He's a good man. And thorough.

**Comment by wireddown** [comment]
Any further harm visited upon Bunny, shall be visited tenfold upon your head. Your goons'll be able to get it off him, mean he's only fifteen and he's flunking social studies.

**Comment by wireddown** [comment]
So if you'll just write me a check for my ten per cent… of half a million… fifty grand.

**Comment by wireddown** [comment]
Her life is in your hands. My point, Dude, is why should we settle for twenty grand when we can keep the entire million. Am I wrong?

**Comment by wireddown** [comment]
Is this yours, Larry? Is this your homework, Larry?

**Comment by wireddown** [comment]
I—the royal we, man, you know, the editorial.

**Comment by wireddown** [comment]
What in God's holy name are you blathering about?

**Comment by wireddown** [comment]
Donny was a good bowler, and a good man. He was… He was one of us.

**Comment by wireddown** [comment]
He was one of us. He was a man who loved the outdoors, and bowling, and as a surfer explored the beaches of southern California from Redondo to Calabassos

**Comment by wireddown** [comment]
nd he was an avid bowler. And a good friend. He died—he died as so many of his generation, before his time.

**Comment by wireddown** [comment]
His girlfriend gafe up her toe! She sought we'd be getting million dollars! Iss not fair!

**Comment by wireddown** [comment]
eg cask conditioned ale dry hopping brewing barley. adjunct, barleywine aerobic krausen chocolate malt wort dextrin, wit goblet.

**Comment by wireddown** [comment]
bittering hops brewing sparge filter hydrometer keg lagering squares original gravity mash tun. real ale malt caramel malt sour/acidic mash lambic, brewing gravity. oxidized pilsner bacterial tulip glass, filter becher. krug; squares acid rest.

**Comment by wireddown** [comment]
noble hops, krausen reinheitsgebot carbonation craft beer. bitter; pint glass anaerobic.

**Comment by wireddown** [comment]
barley carboy.

**Comment by wireddown** [comment]
hydrometer bottom fermenting yeast bunghole, hard cider gravity oxidized dextrin krausen copper; pub black malt bung.

**Comment by wireddown** [comment]
pitching amber, ester bottle conditioning gravity pilsner draft (draught) sparge barrel.

**Comment by wireddown** [comment]
filter cask, hops alcohol biere de garde primary fermentation dextrin top-fermenting yeast.

**Comment by wireddown** [comment]
I hate the Wetlands. They're stupid and wet, and there are bugs everywhere, and I think I maced a crane.

**Comment by wireddown** [comment]
I was once called the worst audience participant Cirque du Soleil ever had.

**Comment by wireddown** [comment]
She calls it a mayonegg.

**Comment by wireddown** [comment]
Michael, I'm your older brother. I'll never be proud of you.

**Comment by wireddown** [comment]
See for more good stuff. I want to cry so bad, but I don't think I can spare the moisture.

**Comment by wireddown** [comment]
Yeah, that's a cultural problem is what it is. You know, your average American male is in a perpetual state of adolescence, you know, arrested development.

**Comment by wireddown** [comment]
Wow. You, sir, are a mouthful! Let me give that oatmeal some brown sugar.

**Comment by wireddown** [comment]
I am going to my spin class. I thought you had vertigo.

**Comment by wireddown** [comment]
I guess you can say I'm buy-curious.

**Comment by wireddown** [comment]
What's gotten into you? Have you been eating cheese?

**Comment by wireddown** [comment]
This is a demo comment.

**Comment by franklinbaldo** [comment]
1115

**Comment by Purpzie** [comment]
Henlo

**Comment by DavidBergeron-eng** [comment]
For the journey has not yet begun it has only been suggested in Lou of change for the light is a spirit that lives and grows its greatest attribute even in darkness; it is has been and always will be greater than the strongest darkness known:CORE ATOM,NEBULA:Core'S children!!!STARS### THE INFINITE,;ELEMENTALL SOURCES DISPERSED!!!
